src/main/java/com/google/devtools/build/lib/rules/objc/XcodeConfig.java

```diff
@@ -397,9 +396,9 @@ private static Map.Entry<XcodeVersionRuleData, Availability> resolveXcodeFromLoc
             String.format(
                 "--xcode_version=%1$s specified, but it is not available locally. Your build will"
                     + " fail if any actions require a local Xcode. If you believe you have '%1$s'"
-                    + " installed, try running \"sudo python3 /usr/local/bin/xcode_configure.py"
-                    + " --verbose\", and then re-run your command. Locally available versions:"
```

Comment:
nit: mind fixing the typo ("localy" -> "locally")

Also below and in the test case.